### PR TITLE
RFC: Qri RFC Process

### PR DESCRIPTION
**Pull request by b5**
proposal 1: define a proposal.

such meta.

### CODE REVIEW

File: text/0001-rfc_process.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+Upon accepting this proposal, we'll also move all of the issues filed in 
+[qri-io/notes](https://github.com/qri-io/notes) & depricate the repo. If we
+accept this RFC process, we should commit to it fully. Notes should now
+start as issues on this repo with the intention that they could develop into 
+formal RFCs.
+
+At the same time, this should also be a chance for others to start documenting
+ideas we've been kicking around for enhancing qri. Things we've discussed in the
+past like like skylark modules, simulation testing, new file format support,
```
Comment (dustmop) [inline text/0001-rfc_process.md]: Double like

File: text/0001-rfc_process.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+[guide-level-explanation]: #guide-level-explanation
+
+For the core team adopting this RFC process will feel like three things at once:
+  - a source of truth for _how qri should behave_
+  - a new channel to directly impact the design of qri
+  - lots of additional work to get new stuff into qri
+
+The first batch of RFCs should ideally come from me (@b5), outlining in detail
+how the various aspects of qri _should_ work (which as we know, is different 
+from how things do/not work). The process of debating these initial RFCS should
```
Comment (dustmop) [inline text/0001-rfc_process.md]: Spelled as "RFCs" earlier in this paragraph, but "RFCS" here.

File: text/0001-rfc_process.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+principle of [rough consensus and working code](https://www.ietf.org/about/participate/tao/).
+
+One of the main concerns I have is ambiguity exists around what is _expected 
+behaviour_ when interacting with qri. To me, this is a _major_ problem. 
+Not knowing precisely what working-as-expected looks like cuts down on what core 
+team members can confidently reason about, which means the core can't
+confidently tell _others_ how qri should work, which means the whole project is
+both ambiguous and without a clear path for resolving that ambiguity.
+
+To me this ambiguity is a sign that we (the core team) haven't taken aren't
```
Comment (dustmop) [inline text/0001-rfc_process.md]: Missing slash between "haven't taken" and "aren't taking"

File: text/0001-rfc_process.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+- Issue: N/A
+
+# Summary
+[summary]: #summary
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for charting the roadmap of qri. We've seen a number of 
+projects in the distributed space suffer from under-considered design choices 
+and unclear roadmapping, We're hoping strong adherence to a lightweight RFC 
+process can help mitigate these problems. Anoyone should be able get a sense of
```
Comment (ramfox) [inline text/0001-rfc_process.md]: "Anoyone should be able get a sense of" -> "Anyone should be able get a sense of "

File: text/0001-rfc_process.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+from how things do/not work). The process of debating these initial RFCS should 
+serve as a chance to work together to establish language, clarity & detail 
+around expected behaviours. Accepting these initial RFCs provides a new,
+consensus-driven foundation for the project that should give core team members
+a confident, consistent source of truth to work from.
+By requiring sign-off from the core team, this should be a chance for the
+core team to achieve consensus on how things should work.
+
+Upon accepting this proposal, we'll also move all of the issues filed in 
+[qri-io/notes](https://github.com/qri-io/notes) & depricate the repo. If we
```
Comment (dustmop) [inline text/0001-rfc_process.md]: 👍

File: text/0001-rfc_process.md [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+and unclear roadmapping, We're hoping strong adherence to a lightweight RFC 
+process can help mitigate these problems. Anoyone should be able get a sense of 
+_where qri is going_ by reading through the accepted proposals.
+
+# Motivation
+[motivation]: #motivation
+
+I openly adknowledge this may seem premature for such an early-stage project.
+I'm intending to put this RFC place in process now to develop a design-driven
+culture that others have a clear path to contribute to the future of the
```
Comment (ramfox) [inline text/0001-rfc_process.md]: "culture that others have" => "culture so that others have

File: text/0001-rfc_process.md (hunk 3 above)
Comment (ramfox) [inline text/0001-rfc_process.md]: "haven't taken aren't taking" => "haven't taken/aren't taking" or just one or the other?

File: text/0001-rfc_process.md [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+    if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+      a *summary comment* trying to lay out the current state of the discussion
+      and major tradeoffs/points of disagreement.
+    - Before actually entering FCP, *all* members of the core team must sign off;
+    this is often the point at which many core team members first review the RFC
+    in full depth.
+  - The FCP lasts ten calendar days, so that it is open for at least 5 business
+    days. It is also advertised widely,
+    e.g. in [This Week in Qri](https://this-week-in-rust.org/). This way all
```
Comment (ramfox) [inline text/0001-rfc_process.md]: This is currently linking to https://this-week-in-rust.org/

Do we have a This Week in Qri? If not, we should have it before this gets merged.

File: text/0001-rfc_process.md [hunk 8]
```diff
@@ -0,0 +1,244 @@
...
+    stakeholders have a chance to lodge any final objections before a decision
+    is reached.
+  - In most cases, the FCP period is quiet, and the RFC is either merged or
+    closed. However, sometimes substantial new arguments or ideas are raised,
+    the FCP is canceled, and the RFC goes back into development mode.
```
Comment (dustmop) [inline text/0001-rfc_process.md]: cancelled

File: text/0001-rfc_process.md (hunk 7 above)
Comment (b5) [reply text/0001-rfc_process.md]: we do not, will remove 😄

**Comment by ramfox** [comment]
Let's merge this :)

🐰 🦊 🐮

**Pull request merged**